src/v2i-hub/ImmediateForwardPlugin/src/ImmediateForwardConfiguration.h

```diff
@@ -49,6 +92,7 @@ namespace ImmediateForward
         tmx::utils::rsu::RSU_SPEC spec;
         std::string address;
         unsigned int port;
+        std::optional<SNMPAuth> snmpAuth;
         std::vector<Message> messages;
```

Comment:
This Message type naming is a bit confusing.

Reply:
For comments where you believe variable naming is confusing please indicate what is confusing and why, other wise these sorts of comments are difficult to address. Adding preferred  alternatives also helps but is not necessary

Reply:
The "Message" type and its message variable have two meanings in this PR context:
- Configuration type: https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/b5deb40a2959aeff56647ddf7fb4eac2f57ae61d/src/v2i-hub/ImmediateForwardPlugin/manifest.json#L18
- A plain string of message: https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/b5deb40a2959aeff56647ddf7fb4eac2f57ae61d/src/v2i-hub/ImmediateForwardPlugin/src/ImmediateForwardPlugin.cpp#L256

Reply:
Will address